bentoml/_internal/bento/build_config.py

```diff
@@ -402,6 +400,8 @@ def write_to_bento(self, bento_fs: "FS", build_ctx: str) -> None:
                 whl_file = resolve_user_filepath(whl_file, build_ctx)
                 copy_file_to_fs_folder(whl_file, bento_fs, wheels_folder)
 
+        build_bentoml_editable_wheel(bento_fs.getsyspath(wheels_folder))
+
```

Comment:
in the `def __attrs_post_init__(self):` of DockerOptions, add a warning for `system_packages`